TUMFARSynchrony/experimental-hub
# Frontend consistent code style

### Description
This PR adds 
1) ESLint(static code analysis tool) - to detect errors and enforce best practices in the code.
2) Prettier - to make sure that the code style is consistent. 
3) Husky - run ESLint before each commit to prevent breaking commits

### Testing
1) run "npm install"
2) Make sure to install the prettier extension in VS Code:
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode
3) Set it as the default formatter in your VS Code settings. This setting can be set for all languages or by a specific language.
4) Format a few files and observe that source control is clear(no changes in terms of code style)
